string name
float32 x_screen
float32 y_screen
